src/library_glfw.js

```diff
@@ -1112,7 +1106,8 @@ var LibraryGLFW = {
       if (!Module.ctx && useWebGL) return 0;
 
       // Get non alive id
-      var win = new GLFW_Window(id, width, height, title, monitor, share);
+      const canvas = Module['canvas'];
+      var win = new GLFW_Window(id, canvas.clientWidth, canvas.clientHeight, canvas.width, canvas.height, title, monitor, share);
```

Comment:
Looks like this changes the semantics of the existing `width` and `height` parameters passed to the `GLFW.createWindow(width, height)`  function, and the `GLFW_Window` ctor:

previously they were the WebGL front buffer pixel dimensions,
now after this PR `width` and `height` instead refer to the visible CSS size of the canvas.

Is that semantical change intentional?

Reply:
Great, sounds good. LGTM.